modules/dynamic/test/src-jvm/smithy4s/dynamic/DynamicJsonProxySpec.scala

```diff
@@ -80,4 +80,15 @@ object DynamicJsonProxySpec extends weaver.SimpleIOSuite {
 
   }
 
+  test("Dynamic service based proxy supports service errors") {
```

Comment:
Your test actually doesn't test this : you should use the knowledge of a json-level transformation to ensure that the dynamic proxy did turn the exception into a piece of json. Otherwise the exception might just have flown through the logic via the monad's error channel, and received by the "static-aware" client.

Reply:
See https://github.com/disneystreaming/smithy4s/pull/216/files#diff-504e8390cf57260069c50d7e97512dae022a6f19248575414777fb0ee6310c4aR66

Reply:
got it, fixed